### PR TITLE
Added hidden tags to remove document and reason links

### PR DESCRIPTION
**Pull request by SVarallo**

### CODE REVIEW

File: views/check-your-answers.html [hunk 1]
```diff
@@ -30,8 +30,6 @@ <h3 class="govuk-heading-m">Reason {{ loop.index }}</h3>
             <dd class="app-check-your-answers__answer">
               {{ reason.reason | capitalize }}
             </dd>
-            <dd class="app-check-your-answers__change">
```
Comment (mattch1) [inline views/check-your-answers.html]: Why were these removed were they just not being used?

Comment (SVarallo) [reply views/check-your-answers.html]: Yeah, they were just empty tags. Figured I'd clear them out.

Comment (mattch1) [reply views/check-your-answers.html]: ok
